internal/ad/ad_test.go

```diff
@@ -626,6 +626,20 @@ func TestGetPolicies(t *testing.T) {
 			turnKrb5CCRO:       true,
 			wantErr:            true,
 		},
+		"Unsupported type for unfiltered entry": {
+			gpoListArgs:        []string{"gpoonly.com", "bob:bad-entry-type"},
```

Comment:
I don’t remember how we did it for the other, but can’t we change the name (a.k.a valueName) of the bad entry itself to identify it in the .pol file?